docs/layers.rst

```diff
@@ -99,3 +99,5 @@ another tensor. DeepChem maintains an extensive collection of layers which perfo
 
 .. autoclass:: deepchem.models.layers.SetGather
   :members:
+
+.. autofunction:: deepchem.models.layers._cosine_dist
```

Comment:
Added function docstring to documentation (I think is what this does...)

Reply:
Yep this works! However, we should rename `_cosine_dist` to `cosine_dist` (in general having a `_` at the front indicates a function is private and not to be used more broadly). Since the implementation is cleaned up, we should make it public so more people can use it :)

Reply:
OK, I changed everywhere it's called to be `cosine_dist` at least in the `layers.py` file. Let me know if there are other locations that need to be changed.